utils/docker-ce/Makefile

```diff
@@ -133,4 +133,21 @@ define Package/docker-ce/install
 		$(1)/etc/sysctl.d/12-br-netfilter-ip.conf
 endef
 
+define Package/docker-ce/postinst
+#!/bin/sh
+[ -n "$$IPKG_INSTROOT" ] || {
+	/etc/init.d/dockerd enable
+	/etc/init.d/dockerd uciadd
+	/etc/init.d/dockerd start
+}
+endef
+
+define Package/docker-ce/prerm
+#!/bin/sh
+[ -n "$$IPKG_INSTROOT" ] || {
+	/etc/init.d/dockerd disable
+	/etc/init.d/dockerd stop
```

Comment:
Perhaps it should be left until later. 
but it'd be nice if we could prompt and run `ucidel` if the user selects `y` or run it by default if there isn't anything in `uci changes`

Reply:
The uci changes are tricky because there are different staging areas for them.
The LuCI uses something different than when we run the uci config via cmd.
If we want to prompt the user if the config should be deleted, then this does not work if we remove the package over the LuCI. So I think the only solution is either do not remove the uci or remove them with out asking as it is now.

Reply:
> remove them with out asking as it is now

Just to clarify, you mean with `ucidel` right? because I can see it on the package removal.

In the future, it might be nice to look into using `uci changes firewall` to detect if we can make changes and commit. because that way we could remove all the docker UCI additions on the package removal.

Reply:
> > remove them with out asking as it is now
> 
> Just to clarify, you mean with `ucidel` right? because I can see it on the package removal.

Yes

> In the future, it might be nice to look into using `uci changes firewall` to detect if we can make changes and commit. because that way we could remove all the docker UCI additions on the package removal.

That's what I would do with the command ucidel?